airbyte-ci/connectors/pipelines/pipelines/helpers/git.py

```diff
@@ -11,11 +11,11 @@
 
 
 def get_current_git_revision() -> str:  # noqa D103
-    return git.Repo().head.object.hexsha
+    return git.Repo(search_parent_directories=True).head.object.hexsha
```

Comment:
What was happening here? Is it something that's happening outside of this branch?

Reply:
Basically its a small improvement where we should always be searching parent repositories.

Just being proactive!

Reply:
Sweet :)